_episodes/00-quickstart.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Quickstart Guide"
+teaching: 5
+exercises: 10
```

Comment:
Will you add `exercises` to this lesson?

Reply:
One suggestion for lesson development in the future: if learners want to practice the skills with the exercises, there should be a solution/answer. For example, when we asked them to run `esmvaltool --help`, we should also tell them what would be the output. More examples can be found in other episodes such as [03-configuration](https://esmvalgroup.github.io/ESMValTool_Tutorial/03-configuration/index.html). Also for more information see [Designing challenges](https://carpentries.github.io/curriculum-development/designing-challenges.html#designing-challenges-1).